Resources/views/Form/iconpicker.html.twig

```diff
@@ -0,0 +1,34 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+{% block sonata_type_icon_selector_widget %}
+    {{ block('choice_widget') }}
+    {% spaceless %}
+        <script type="text/javascript">
+            jQuery(function ($) {
+                var select2FormatColorSelect = function format(state) {
+                    if (!state.id || state.disabled) {
+                        return state.text;
+                    }
+
+                    return ' <i class="fa '+ state.text +'"></i> ' + state.text;
```

Comment:
This code is currently used in SonataCoreBundle in colorpicker.html.twig.
BTW, there is a bug in the javascript so that the color square do not appear (symfony 2.8.2)
Maybe you should correct the color widget in SonataCoreBundle first ?